src/js/control-bar/time-controls/time-display.js

```diff
@@ -51,6 +51,10 @@ class TimeDisplay extends Component {
 
     this.updateTextNode_();
     el.appendChild(this.contentEl_);
+    if (className === 'vjs-remaining-time') {
```

Comment:
this might be better to do inside of remaining-time-display itself. We can override `createEl()`, call `super.createEl()` in it, then insert this span there before returning the el. That will keep all the remaining time display stuff together.

Reply:
@gkatsev In my last commit i followed your suggestion and created a createEl for the remaining-time-display. I also deleted the function formatTime on that file as it isn't being used anymore